screenpy/actions/make_note.py

```diff
@@ -56,6 +57,7 @@ def of_the(cls, question: T_Q) -> Self:
     def as_(self, key: str) -> Self:
         """Set the key to use to recall this noted value."""
         self.key = key
+        self.key_to_log = represent_prop(key)
```

Comment:
This make sense, but I'm starting to wonder if we should consider a wholesale update to any action similar to this.
I think we ought to make `key_to_log` a property.

Reply:
That's a good call, and yeah let's make that happen soon.